packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/component.js

```diff
@@ -1,8 +1,9 @@
-import controller from './controller';
 import template from './template.html';
 
 const component = {
-  controller,
+  bindings: {
+    stein: '<',
```

Comment:
What does `stein` mean ? Is it a boolean ? In that case maybe `isSteinRelease` ?

Reply:
it's name used by API.